tests/test_sanitize.py

```diff
@@ -33,6 +34,24 @@ def test_strip_html(self):
             sanitize.strip_html('<foo>bar</foo>'),
             'bar'
         )
+        assert_equal(
+            sanitize.strip_html(b'<foo>bar</foo>'),
+            'bar'
+        )
+
+    def test_strip_html_on_non_strings_returns_original_value(self):
+        assert_true(sanitize.strip_html(True))
+        assert_false(sanitize.strip_html(False))
+
+        assert_equal(sanitize.strip_html(12), 12)
+        assert_equal(sanitize.strip_html(12.3), 12.3)
+
+        dtime = datetime.datetime.now()
+        assert_equal(sanitize.strip_html(dtime), dtime)
+
+    def test_strip_html_bleaches_collection_types(self):
```

Comment:
We may replace bleach in the future, so try to avoid hard-coding references to the tool (instead of the act of sanitizing) when writing test names.

Eg `test_strip_html_sanitizes_collections_as_strings`